src/collections/posts/comfy-sabers.md

```diff
@@ -3,8 +3,9 @@ title: Comfy Sabers
 section: articles
 category: event
 publish: 2024-04-24T19:00:00.000Z
+homepageText: Event landing page for Comfy Sabers - updated as tournaments become available.
```

Comment:
This seems odd for a homepageText. Is this that intended? I would rather expect something that introduces the article without telling to mouch.

Reply:
I suppose I could probably write up a better homepage text. My goal originally was to convey that the page would be updated regularly (or as tournaments become available) but I suppose if it’s going to be updated regularly anyways I could just update that as needed, too.